client/src/components/History/Content/GenericElement.vue

```diff
@@ -1,23 +1,30 @@
 <script setup lang="ts">
-import { type PropType, ref, set } from "vue";
+import { type PropType, type Ref, ref, set } from "vue";
 
 import { type components } from "@/api/schema";
 
 import ContentItem from "./ContentItem.vue";
 
+type ElementObject = components["schemas"]["DCObject"];
+type Element = components["schemas"]["DCESummary"];
```

Comment:
These type aliases already live in "@/api" I would suggest importing them from there instead of directly using `components`. But it's not a big deal.

Reply:
Done!